src/main/main.js

```diff
@@ -554,4 +559,21 @@ ipcMain.on('get-auth-token', (event) => {
 
 ipcMain.on('set-auth-token', (event, token) => {
   keytar.setPassword("LBRY", "auth_token", token ? token.toString().trim() : null);
-});
\ No newline at end of file
+});
+
+autoUpdater.logger = log;
+autoUpdater.on('checking-for-update', () => {
```

Comment:
It might make sense to split the update logic into it's own file if it's not interacting with `main.js` much. `main.js` is already a little big and cluttered.